features/configuration_files/directory_specific_directives.feature

```diff
@@ -0,0 +1,171 @@
+Feature: Directory directives
```

Comment:
Given our recent discussion about Cucumber, I think this should end up being the only configuration scenario, because it is the most complex example.